scripts/sync-kernel.sh

```diff
@@ -45,7 +45,6 @@ PATH_MAP=(									\
 	[tools/include/uapi/linux/if_link.h]=include/uapi/linux/if_link.h	\
 	[tools/include/uapi/linux/if_xdp.h]=include/uapi/linux/if_xdp.h		\
 	[tools/include/uapi/linux/netlink.h]=include/uapi/linux/netlink.h	\
-	[tools/include/tools/libc_compat.h]=include/tools/libc_compat.h		\
```

Comment:
can you please remove this file from include/tools as well?

Reply:
I did remove `include/tools/libc_compat.h`... so you missed that or I don't understand?

Reply:
ah, my bad, missed it in "Files changed" view, sorry about that

Reply:
No problem :-)